todo_app/app.py

```diff
@@ -1,11 +1,31 @@
-from flask import Flask
-
+#mydict = dict((rows[0],rows[1]) for rows in reader)
+from flask import Flask, render_template, redirect, request
+from todo_app.data.session_items import get_items, add_item
```

Comment:
This file is still using all your old functionality, but rather than that we want to move our entire app over to use Trello; we can always recover your M1 work using Git at a later date. Because of that, we should bring the Flask functionality from trello_items into this file, and just import the Trello specific logic (like getting & creating cards) from the trello_items file

This is important because now your repository is storing multiple different apps, and your Readme instructions are totally out of date for running the Trello version of the app so new users are going to be very confused on how to run this project or even which app is the right one

Reply:
I have moved the trello logic to be included in the app.py.